homeassistant/components/renson_endura_delta/const.py

```diff
@@ -0,0 +1,218 @@
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TIMESTAMP,
+    TEMP_CELSIUS,
+)
+
+DOMAIN = "renson_endura_delta"
+CONCENTRATION_PARTS_PER_CUBIC_METER = "ppm"
+
+CO2_QUALITY_DESC = SensorEntityDescription(
+    key="CO2_QUALITY_FIELD", name="CO2 quality", state_class=STATE_CLASS_MEASUREMENT
+)
+
+CO2_DESC = SensorEntityDescription(
+    key="CO2_FIELD",
+    name="CO2 quality value",
+    device_class="carbon_dioxide",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=CONCENTRATION_PARTS_PER_CUBIC_METER,
+)
+
+AIR_QUALITY_DESC = SensorEntityDescription(
+    key="AIR_QUALITY_FIELD", name="Air quality", state_class=STATE_CLASS_MEASUREMENT
+)
+
+AIR_DESC = SensorEntityDescription(
+    key="AIR_FIELD",
+    name="Air quality value",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=CONCENTRATION_PARTS_PER_CUBIC_METER,
+)
+
+CURRENT_LEVEL_RAW_DESC = SensorEntityDescription(
+    key="CURRENT_LEVEL_FIELD_RAW",
+    name="Ventilation level raw",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+CURRENT_LEVEL_DESC = SensorEntityDescription(
+    key="CURRENT_LEVEL_FIELD",
+    name="Ventilation level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+CURRENT_AIRFLOW_EXTRACT_DESC = SensorEntityDescription(
+    key="CURRENT_AIRFLOW_EXTRACT_FIELD",
+    name="Total airflow out",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="m³/h",
+)
+
+CURRENT_AIRFLOW_INGOING_DESC = SensorEntityDescription(
+    key="CURRENT_AIRFLOW_INGOING_FIELD",
+    name="Total airflow int",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="m³/h",
+)
+
+OUTDOOR_TEMP_DESC = SensorEntityDescription(
+    key="OUTDOOR_TEMP_FIELD",
+    name="Outdoor air temperature",
+    device_class="temperature",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=TEMP_CELSIUS,
+)
+
+INDOOR_TEMP_DESC = SensorEntityDescription(
+    key="INDOOR_TEMP_FIELD",
+    name="Extract air temperature",
+    device_class="temperature",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=TEMP_CELSIUS,
+)
+
+FILTER_REMAIN_DESC = SensorEntityDescription(
+    key="FILTER_REMAIN_FIELD",
+    name="Filter change",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="days",
+)
+
+HUMIDITY_DESC = SensorEntityDescription(
+    key="HUMIDITY_FIELD",
+    name="Relative humidity",
+    device_class=DEVICE_CLASS_HUMIDITY,
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="%",
+)
+
+MANUAL_LEVEL_DESC = SensorEntityDescription(
+    key="MANUAL_LEVEL_FIELD",
+    name="Manual level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+TIME_AND_DATE_DESC = SensorEntityDescription(
+    key="MANUAL_LEVEL_FIELD",
+    name="System time",
+    device_class=DEVICE_CLASS_TIMESTAMP,
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+BREEZE_TEMPERATURE_DESC = SensorEntityDescription(
+    key="BREEZE_TEMPERATURE_FIELD",
+    name="Breeze temperature",
```

Comment:
This is configuration of what temperature "Breeze" mode is set to? 

- Disable it by default by setting `entity_registry_enabled_default=False`
- Set entity category to `diagnostic`

Do this for all other config related entities here too